core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -25,6 +25,99 @@
   {{dependencies_html}}
 {% endblock header_js %}
 
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak" ng-controller="CollectionEditorNavbarBreadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span ng-if="collection">
+        <[collection.getTitle()]>
+        <span class="oppia-navbar-breadcrumb-separator" style="margin-left: 5px;"></span>
+        <span>
+          <[getCurrentTabName()]>
+        </span>
+      </span>
+      <span ng-if="!collection">
+        Loading...
+      </span>
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block local_top_nav_options %}
+  <ul class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak" ng-controller="CollectionEditorNavigation" style="margin-left: 20px;">
+    <!-- Editor -->
+    {% if SHOW_COLLECTION_NAVIGATION_TAB_MAIN %}
+    <li ng-class="{'active': getTabStatuses().active === 'main', 'dropdown': countWarnings()}">
+      <a href="#" tooltip="Editor" tooltip-placement="<[countWarnings() ? 'left' : 'bottom']>" ng-click="selectMainTab()" class="protractor-test-main-tab">
+        <span class="glyphicon glyphicon-pencil"></span>
+      </a>
+    </li>
+    {% endif %}
+
+    <!-- Preview -->
+    {% if SHOW_COLLECTION_NAVIGATION_TAB_PREVIEW %}
```

Comment:
You can also do ``{% if SHOW_COLLECTION_NAVIGATION_TAB_PREVIEW and username %}``